### DISCUSSION TITLE
How to upgrade Gateway API - is documentation missing?

### DISCUSSION THREAD

**Discussion by pleshakov** [kubernetes-sigs/gateway-api#2227] [Ideas]
The [getting starting guide](https://gateway-api.sigs.k8s.io/guides/#getting-started-with-gateway-api) includes instructions about how to install the Gateway API and how to uninstall it. However, there is no instructions about how to upgrade the Gateway API from one version to the next.

Would it make sense to have some documentation about that?

**Comment by robscott** [comment]
+1 that's a great idea. I think for now it's pretty straightforward, but if we ever drop an API version, we'll need to reference something like [kube-storage-version-migrator](https://github.com/kubernetes-sigs/kube-storage-version-migrator)
